pom.xml

```diff
@@ -58,6 +58,11 @@
             <artifactId>log4j</artifactId>
             <version>1.2.16</version>
         </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20180130</version>
+        </dependency>
     </dependencies>
```

Comment:
if you can show me how to use it, it'll be a great time saving for me :)

Reply:
We'll do it in the UI and upload an example :).